Rakefile

```diff
@@ -17,13 +18,9 @@ HOE = Hoe.spec "puma" do
 
   spec_extras[:extensions]  = ["ext/puma_http11/extconf.rb"]
   spec_extras[:executables] = ['puma', 'pumactl']
-  spec_extras[:homepage] = self.urls.first
+  spec_extras[:homepage] = urls.first
 
   require_ruby_version ">= 1.9.3"
-
-  dependency "rack", [">= 1.1", "< 3.0"], :development
-
-  extra_dev_deps << ["rake-compiler", "~> 0.8"]
```

Comment:
these do not match what is in the Gemfile and used locally or travis ... useless anyway so just removing them ...